mk/docs.mk

```diff
@@ -126,6 +126,7 @@ doc/not_found.html: $(D)/not_found.md $(HTML_DEPS) | doc/
 		--markdown-css https://doc.rust-lang.org/rust.css $<
 
 define DEF_DOC
+RUSTDOC_DEPS_$(1) ?=
```

Comment:
These guys did not specifically uncover bugs, and I agree with you that sprinkling them around is gross. The value of adding these is that it suppresses some warnings; in a world where there are no spurious warnings, people are incented to investigate each warning they introduce.

That said, we are still a long way from that, so I'll back this out if you insist.